llvm/test/Analysis/UniformityAnalysis/AMDGPU/always_uniform.ll

```diff
@@ -58,6 +67,18 @@ define void @single_lane_func_arguments(i32 %i32, i1 %i1) #2 {
  ret void
 }
 
+; CHECK-LABEL: for function 'divergent_args':
+; CHECK: DIVERGENT ARGUMENTS
```

Comment:
don't need ARGUMENTS?

Reply:
Here it's a "CHECK", so the longer string "DIVERGENT ARGUMENTS" is a stronger condition than just "DIVERGENT". Therefore I think we can leave it as it is.